README.md

```diff
@@ -4,7 +4,7 @@
 # Blog-App
 
 ## Description 🏗️
-I will add forms to allow users to add comments and likes to posts on behalf of the correct user
+I will add integration specs and fix n+1 problem
 
```

Comment:
- [ ] Kindly organize your files. Separate the tests for the Posts Show page and the Page Index page into different files.

- [ ] For the Posts index page, you are missing tests for the following assertions. Kindly add them.

  * Tests that asserts you can see a post's title.
  * Tests that asserts you can see how many comments a post has.
  * Tests that asserts you can see how many likes a post has.
  * Tests that asserts you can see a section for pagination if there are more posts than fit on the view.
  * Tests that redirects to the post's show page when you click on a post.

- [ ] For the User show page you are missing the following assertions. Kindly add them.

  * Tests that asserts you can see the post's title.
  * Tests that asserts you can see the post body.
  * Tests that asserts you can see the username of each commentor.
  * Tests that asserts you can see the comment each commentor left.

Reply:
@TedLivist 
![image](https://user-images.githubusercontent.com/30408871/200321325-48f59efc-28be-4732-bc79-3fd2d7cd3dee.png)
I ran them just now and they all passed. 

All of the test cases that you are asking me to re-run are passing apart from the few I have not implemented like first three posts etc.